src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingSymbolTranslator.cs

```diff
@@ -57,9 +57,29 @@ private RetargetingNamedTypeSymbol CreateRetargetingNamedType(Symbol symbol)
             return new RetargetingNamedTypeSymbol(this, (NamedTypeSymbol)symbol);
         }
 
-        private RetargetingFieldSymbol CreateRetargetingField(Symbol symbol)
+        private FieldSymbol CreateRetargetingField(Symbol symbol)
         {
             Debug.Assert(ReferenceEquals(symbol.ContainingModule, _underlyingModule));
+            if (symbol is TupleErrorFieldSymbol tupleErrorField)
+            {
+                var correspondingTupleField = tupleErrorField.CorrespondingTupleField;
+                Debug.Assert(correspondingTupleField is TupleErrorFieldSymbol);
+
+                var retargetedCorrespondingDefaultFieldOpt = (correspondingTupleField == (object)tupleErrorField)
+                    ? null
+                    : (TupleErrorFieldSymbol)RetargetingTranslator.Retarget(correspondingTupleField);
+
+                return new TupleErrorFieldSymbol(
+                    RetargetingTranslator.Retarget(tupleErrorField.ContainingType, RetargetOptions.RetargetPrimitiveTypesByName),
+                    tupleErrorField.Name,
+                    tupleErrorField.TupleElementIndex,
+                    tupleErrorField.Locations.IsEmpty ? null : tupleErrorField.Locations[0],
+                    this.RetargetingTranslator.Retarget(tupleErrorField.TypeWithAnnotations, RetargetOptions.RetargetPrimitiveTypesByName),
```

Comment:
>RetargetPrimitiveTypesByName [](start = 109, length = 28)

Why is this changed? This looks wrong.

Reply:
Sorry, I assumed you meant both places.

---
In reply to: [583737556](https://github.com/dotnet/roslyn/pull/44231#discussion_r583737556) [](ancestors = 583737556)